Code/Tools/AssetProcessor/native/tests/assetmanager/JobDependencySubIdTests.h

```diff
@@ -42,7 +42,7 @@ namespace UnitTests
         void CheckJobEntries(int count);
     };
 
-    struct JobDependencySubIdTest : UnitTest::ScopedAllocatorSetupFixture
+    struct JobDependencySubIdTest : ::UnitTest::ScopedAllocatorSetupFixture
```

Comment:
This isn't important for this PR, but I keep having to randomly add :: for unit tests as well and wonder what's causing this.

Reply:
I removed some headers so I assume there is something that normally conflicts in Uber builds if that header is not there. I am not sure which header did it though.